gdk/Gdk.metadata

```diff
@@ -205,7 +205,14 @@
   <attr path="/api/namespace/struct[@cname='GdkRgbCmap']/field[@name='NColors']" name="writeable">false</attr>
   <attr path="/api/namespace/struct[@cname='GdkTimeCoord']/field[@cname='axes']" name="array_len">128</attr>
   <attr path="/api/namespace/struct[@cname='GdkWindowObject']" name="hidden">1</attr>
+  <attr path="/api/namespace/struct[@cname='GdkPixbufFormat']/method[@cname='gdk_pixbuf_format_get_extensions']/return-type" name="owned">true</attr>
+  <attr path="/api/namespace/struct[@cname='GdkPixbufFormat']/method[@cname='gdk_pixbuf_format_get_mime_types ']/return-type" name="owned">true</attr>
   <add-node path="/api"><symbol type="manual" cname="cairo_t" name="Cairo.Context"/></add-node>
   <add-node path="/api"><symbol type="manual" cname="cairo_font_options_t" name="Cairo.FontOptions"/></add-node>
+  <add-node path="/api/namespace/object[@cname='GdkWindow']">
```

Comment:
This should've been a separate commit. This is the proper fix to applying the correct gtype to gdkwindow, rather than modifying the raw file.